### ISSUE TITLE
First query not available during run time

### ISSUE THREAD

**Issue by jmolivas** [remorses/genql#105]
We are currently migrating to v3 and found an issue, related to first query not available during run time

We workaround it by adding a new model named `Ab` to continue building the app:
![image](https://user-images.githubusercontent.com/366275/219880659-7e64e762-4a29-4add-bd06-c72fd31743f0.png)

I can confirm query is available while writing code: 
![image](https://user-images.githubusercontent.com/366275/219880474-e7f207ad-d32f-4164-af3f-f206214f57ea.png)

But erroring on runtime:
![image](https://user-images.githubusercontent.com/366275/219880540-5fad8fd5-9a46-4e62-a1ee-9df65892a866.png)

![image](https://user-images.githubusercontent.com/366275/219880528-2bf75b3f-5cd6-4533-bb08-a8777bc4a2df.png)

**Comment by remorses** [comment]
Can you paste here your query type? Even the first few lines should be enough

**Comment by jmolivas** [comment]
Hi @remorses is this what you asked for?

```
findFirstAb(cursor: AbWhereUniqueInput, distinct: [AbScalarFieldEnum!], orderBy: [AbOrderByWithRelationInput!], skip: Int, take: Int, where: AbWhereInput): Ab
```

**Comment by remorses** [comment]
Can you paste the whole schema? i need to see where the failing query is defined

I thought you meant the first field defined in `Query` was failing, what do you mean with first?

**Comment by remorses** [comment]
If you can create a reproduction repository that would be perfect

**Comment by Julien-R44** [comment]
Yo, I had the same problem just now, I opened a PR for the same #106

**Comment by remorses** [comment]
Fixed in `3.0.3`

**Comment by jmolivas** [comment]
Thank your both @remorses & @Julien-R44
